metaflow/plugins/kfp/tests/sample_flows/merge_artifacts.py

```diff
@@ -0,0 +1,81 @@
+from metaflow import FlowSpec, step
+
+import pytest
+
+
+class MergeArtifacts(FlowSpec):
```

Comment:
please add a `""" """` comment here like the ones above.
also, can merge artifact testing be "merged" into `ForeachSplitLinear`, to reduce the number of KFP flows we have to run?

Reply:
I'd prefer if `merge_artifacts` is separate because it's testing a different use case. Also, the test only takes ~45 seconds to run, so it's not a large overhead.